meta/Meta.cpp

```diff
@@ -7446,18 +7446,25 @@ void Meta::meta_sai_on_fdb_event_single(
                 sai_attribute_t *list = data.attr;
                 uint32_t count = data.attr_count;
 
-                sai_attribute_t local[2]; // 2 for port id and type
-
-                if (count == 1)
+                /* Add an attr of type if there is no SAI_FDB_ENTRY_ATTR_TYPE in data.
+                 * There are two attributes but no SAI_FDB_ENTRY_ATTR_TYPE in VXLAN environment.
+                 */
+                sai_attribute_t local[SAI_FDB_ENTRY_ATTR_END];
```

Comment:
count possibly can be greater than local array, please use std::vector<sai_attribute_t>